src/core/src/Environment.C

```diff
@@ -112,6 +112,10 @@ FilePtrSetStruct::FilePtrSetStruct()
   :
   ofsVar(NULL),
   ifsVar(NULL)
+#ifdef QUESO_HAS_HDF5
+  ,  // lol
```

Comment:
If QUESO has HDF5, this code is compiled.  Without the comma, the previous line does not have a comma and you get a compiler error.  If QUESO does not have HDF5, the previous line is the last line in the initialisation list and therefore cannot have a comma.

"Lol" means "laughing out loud", and it is there because a single line with a comma amused me greatly.